core/capabilities/ccip/ccipevm/gas_helpers.go

```diff
@@ -51,18 +51,27 @@ func bytesForMsgTokens(numTokens int) int {
 }
 
 // CalculateMessageMaxGas computes the maximum gas overhead for a message.
+// TODO: update interface to return error
```

Comment:
Might be worth updating now to simplify migration in the future. You'd have to come up with a second function name, which isn't ideal.

```
func (gp EstimateProvider) CalculateMessageMaxGas(msg cciptypes.Message) uint64 {
    max, _ := gp.CalculateMessageMaxGasWithError(msg)
    return max
}
```

Reply:
Agree, if it's simple maybe it's gonna better to it right now. If not then leave a ticket in our backlog, it's easier to track it down if keep it in Jira as well